src/components/wrappers/GridWrapper.ts

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components/macro";
+import { grid, GridProps, compose } from "styled-system";
+
+import { tablet } from "styles/breakpoints";
+import { Box } from "./Box";
+import { Theme } from "styles/theme";
+
+interface GridWrapperProps<T> extends GridProps<T> {}
+
+const gridProps = compose(grid);
+
+export const GridWrapper = styled(Box)<GridWrapperProps<Theme>>`
+  display: grid;
```

Comment:
there's no need for fixed paddings and columns. GridWrapper should be universal component. in this case it has custom static properties. this won't be reusable everywhere